.github/workflows/build-book.yaml

```diff
@@ -75,7 +75,6 @@ jobs:
         uses: dawidd6/action-download-artifact@v3.1.4
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          workflow: trigger-book-build.yaml
```

Comment:
I think this line is needed to identify the workflow that produced the artifact.

Reply:
Okay, so it doesn't start a new action. On the portal site, this isn't a workflow though. The workflow that produces the artifact will be in `publish-site.yaml`. I'll try changing this to be an optional input with the trigger-book-build.yaml as its default value?

Reply:
That explains why the [error on my fork](https://github.com/jukent/projectpythia.github.io/actions/runs/8382046266/job/22955021220) say "Error: no matching workflow run found with any artifacts?"

Reply:
Any other questions I have about the inputs to this step can be answered in the [`action-download-artifact` documentation](https://github.com/dawidd6/action-download-artifact) 🤦‍♀️